config/config.exs

```diff
@@ -1,6 +1,26 @@
 import Config
 
-config :wanda, Wanda.Facts.Messenger, adapter: Wanda.Facts.Messenger.RabbitMQ
+config :wanda, Wanda.Messaging.Publisher, adapter: Wanda.Messaging.Adapters.AMQP
+
+config :wanda, Wanda.Facts.Messenger.Supervisor,
+  children: [Wanda.Messaging.Adapters.AMQP.Publisher, Wanda.Messaging.Adapters.AMQP.Consumer]
+
+config :wanda, :messaging,
+  adapter: Wanda.Messaging.Adapters.AMQP,
+  amqp: [
+    consumer: [
+      queue: "events_wanda",
+      exchange: "events",
```

Comment:
Why do we need an exchange in the consumer?

Reply:
Yes we need to bind the queue with the exchange if it doesn't exist the first time we start the application.